docs/source/new-docs/getting-started/system-requirements/index.rst

```diff
@@ -3,30 +3,71 @@
 System Requirements
 ===================
 
-Assigned to: Jeremy
-Due for review: June 3rd
+GeoIPS system requirements are strongly dependent on how you intend to use
+GeoIPS. This document will outline the minimum and recommended system
```

Comment:
Given that this is a very short document that is less than a few pages, there's no reason to give this one liner here that summarizes the page. It would take user a bit longer to read this sentence than it would take to scroll down and see the section titles.

Reply:
Updated!